### PR TITLE
A simple Kind/Kubernetes-based e2e test

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

File: e2e/verifier-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: verifier
+data:
+  COORDINATOR_API_URL: 'http://coordinator-service:8080'
+  COORDINATOR_AUTH_TYPE: celo
+  COORDINATOR_SEED_FILE: '/mnt/secrets/seed'
```
Comment (kobigurk) [inline e2e/verifier-configmap.yaml:8]: Verifier doesn't need a seed.

**Review by kobigurk — CHANGES_REQUESTED** [review]
Looks great! Just a small comment.

File: e2e/verifier-configmap.yaml (hunk 1 above)
Comment (silasbw) [reply e2e/verifier-configmap.yaml:8]: Oh right.  That's a pre-existing oops with the coordinator-client.  I'll spin up another PR to address that and then circle back around to this one.

Comment (silasbw) [reply e2e/verifier-configmap.yaml:8]: FYI, https://github.com/celo-org/snark-setup-coordinator/pull/6

**Review by kobigurk — APPROVED** [review]
LGTM!

**Pull request merged**